themes/default/content/templates/kubernetes/aws/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Kubernetes Cluster on AWS"
+layout: template
+
+# Make sure this is description accurate for this template.
+meta_desc: The AWS Kubernetes template makes it easy to deploy a Kubernetes cluster on AWS with Pulumi and Amazon EKS.
+
+# Appears on the cards on template-overview pages.
+card_desc: Deploy a Kubernetes cluster on AWS with Pulumi and Amazon EKS.
+
+# Used for generating language-specific links to templates on GitHub. (Example: `static-website-aws`)
+template:
+    prefix: kubernetes-aws
+
+# Used for generating links to sibling templates in the right-hand nav. Slug is this template's parent directory.
+cloud:
+  name: Amazon Web Services
+  slug: aws
+
+# Be sure to replace this image. Figma source file:
+# https://www.figma.com/file/lGrSpwbGGmbixEuewMbtkh/Template-Architecture-Diagrams?node-id=15%3A196
+meta_image: meta.png
+
+# The content below is meant help you get started and to serve as a guide to work by. Feel free to adjust it needed for your template.
+---
+
+The AWS Kubernetes template deploys a managed Kubernetes cluster on AWS with Pulumi infrastructure as code. It deploys an [Amazon EKS cluster]({{< relref "/registry/packages/eks/api-docs/cluster" >}}) that provides a managed Kubernetes control plane and nodes. The nodes run on an [Amazon EC2 instance]({{< relref "/registry/packages/awsx/api-docs/ec2" >}}) connected to a private subnet in a [VPC]({{< relref "/registry/packages/awsx/api-docs/ec2/vpc" >}}). The template also deploys Internet-facing load balancers connected to public subnets. The template generates a complete infrastructure as code program to give you a working project out of the box that you can customize easily and extend to suit your needs.
```

Comment:
I would suggest re-wording like this:

The AWS Kubernetes template deploys a managed Kubernetes cluster on AWS with Pulumi infrastructure as code. This template creates a VPC with both public and private subnets, and then deploys an Amazon EKS cluster that provides a managed Kubernetes control plane. Kubernetes worker nodes are deployed on private subnets for improved security. Load balancers created by workloads deployed on the EKS cluster will be automatically created in the public subnets. The template generates a complete infrastructure as code program to give you a working project out of the box, but which you can customize easily and extend to suit your needs.

Reply:
Obviously you can insert links as needed. Since we don't actually use the AWSx EC2 module, I'm not sure there's any benefit to trying to link that from here.